### ISSUE TITLE
Accessibility issues with media pickers

### ISSUE THREAD

**Issue by nbradbury** [wordpress-mobile/WordPress-Android#21288]
As mentioned [here](https://github.com/wordpress-mobile/WordPress-Android/pull/21287#pullrequestreview-2354911185):

> I note that when a search field is focused, a screen reader is unable to access the network offline message. One can only move focus to the visible illustration, which includes the aforementioned odd alt text. I.e., when the search field is focused, it is not clear at all that a lack of network connection will disrupt the experience. If one dismisses the virtual keyboard, then you can move screen reader focus to the error message and retry button. This experience is present within the GIF library, free stock media, and editor media insert flows.

This is related to the following screens:

- GifMediaDataSource - adding to media library from Tenor
- StockMediaDataSource - adding to media library from free photo library
- MediaLibraryDataSource - adding media to a post from WP media library

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by Siphompeks** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
